flytekit/clients/raw.py

```diff
@@ -264,7 +265,7 @@ def _refresh_credentials_from_command(self):
         :return:
         """
 
-        command = self._cfg.command
+        command = self._cfg.command or Credentials.COMMAND.read()
```

Comment:
We shouldn't need this after https://github.com/flyteorg/flytekit/pull/1063/. 

If you follow click's code, `self._cfg` in the case of `flyte-cli` commands is set [here](https://github.com/flyteorg/flytekit/blob/master/flytekit/clis/flyte_cli/main.py#L514). Notice the use of `PlatformConfig.auto()`, this should [take care](https://sourcegraph.com/github.com/flyteorg/flytekit@87d13903993d5339723ecd859bd362a64e780aec/-/blob/flytekit/configuration/__init__.py?L333-346) of loading the config values using env vars if those are set.

Reply:
Ah ok sorry, missed that